packages/vbulletin.php

```diff
@@ -940,8 +940,8 @@ public function exportMedia($minDiscussionID = false) {
         if ($ex->exists('attachment', array('contenttypeid', 'contentid')) === true) {
             // Exporting 4.x with 'filedata' table.
             // Build an index to join on.
-            $result = $ex->query('show index from :_thread where Key_name = "ix_thread_firstpostid"');
-            if (!$result) {
+            $result = $ex->query('show index from :_thread where Key_name = "ix_thread_firstpostid"', true);
+            if (!mysql_num_rows($result)) {
```

Comment:
If the ExportModel is abstracting away the mysql_ calls, shouldn't it also abstract away counting the rows? Or has that ship sailed, since we call mysql_ functions 9x in this file already?